packages/launcher-component/src/flows/create-new-app-flow.tsx

```diff
@@ -25,13 +25,25 @@ const defaultCustomApp = {
   backend: defaultBackendFormValue,
   frontend: defaultFrontendFormValue,
   destRepository: {
-    repository: {name: generate().dashed}
+    repository: { name: generate().dashed }
   },
   deployment: defaultDeploymentFormValue,
 };
 
 export function CreateNewAppFlow(props: { onCancel?: () => void }) {
-  const [app, setApp] = useState<CustomApp>(defaultCustomApp);
+  const [_app, _setApp] = useLocalStorage('app', JSON.stringify(defaultCustomApp));
+  const app: CustomApp = JSON.parse(_app);
+  const setApp = (value: CustomApp) => {
+    _setApp(JSON.stringify(value));
```

Comment:
@edewit isn't it possible to make the `JSON.stringify` and `JSON.parse` part of the `useLocalStorage`?